lib/datalink.coffee

```diff
@@ -71,7 +71,10 @@ class Datalink
     }, (err, response) ->
       if err? then req.emit 'error', err # Trigger request error.
       else # OK.
-        logger.info 'Deploy initiated, received job %s', chalk.cyan response.body.job # Debug.
+        jobResultId = response.body.job
```

Comment:
you probably don't need this variable at all.
`project.lastJobId = response.body.job` should suffice, no?